modules/siddhi-core/src/main/java/io/siddhi/core/query/selector/QuerySelector.java

```diff
@@ -98,6 +98,23 @@ public void process(ComplexEventChunk complexEventChunk) {
         }
     }
 
+    public void processOptimisedQueryEvents(ComplexEventChunk complexEventChunk) {
```

Comment:
Use consistent naming it should be  `executeOptimisedQueryEvents`
Or better, if we can call it `executePassThrough` if that's what we are actually doing

Reply:
Yes, its passthrough with only current and expired events filtering, renamed as executePassThrough() in 348a2be